subsys/zigbee/lib/zigbee_fota/Kconfig

```
@@ -0,0 +1,83 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig ZIGBEE_FOTA
+	bool "Enable Zigbee FOTA"
+	imply BOOTLOADER_MCUBOOT
+	imply MCUMGR
+	imply DFU_TARGET
+
+if ZIGBEE_FOTA
+
+config ZIGBEE_FOTA_HW_VERSION
+	int "Zigbee hardware version"
+	range 0 65535
+	default 52
+
+config ZIGBEE_FOTA_DATA_BLOCK_SIZE
+	int "Maximum data size of Query Block Image"
+	default 64
+	range 4 64
+
+config ZIGBEE_FOTA_ENDPOINT
+	int "Zigbee OTA endpoint"
+	default 10
+	range 1 240
+
+config ZIGBEE_FOTA_PROGRESS_EVT
+	bool "Emit progress event upon receiving a download fragment"
+
+config ZIGBEE_FOTA_MANUFACTURER_ID
+	hex "Manufacturer ID"
+	default 0x127F
+	range 0x0000 0xFFFF
+	help
+	  This is the ZigBee assigned identifier for each member company.
+	  0x127F - Nordic Semiconductor
+	  0xFFFF - wild card value has a 'match all' effect.
+
+config ZIGBEE_FOTA_IMAGE_TYPE
+	hex "Image type"
+	default 0xFFFF
+	range 0x0000 0xFFFF
+	help
+	  0x0000 - 0xFFBF Manufacturer Specific
+	  0xFFC0 - Client Security credentials
+	  0xFFC1 - Client Configuration
+	  0xFFC2 - Server Log
+	  0xFFC3 - Picture
+	  0xFFFF - Wild card value has a 'match all' effect.
+
+config ZIGBEE_FOTA_COMMENT
+	string "OTA header string"
+	default ""
+	help
+	  Firmware comment to be used in Zigbee OTA header.
+
+config ENABLE_ZIGBEE_FOTA_MIN_HW_VERSION
+	bool "Enable Zigbee OTA minimum hw version"
+
+config ZIGBEE_FOTA_MIN_HW_VERSION
+	hex "Zigbee OTA minimum hw version"
+	default 0
+	range 0x00 0xFF
+	depends on ENABLE_ZIGBEE_FOTA_MIN_HW_VERSION
+
+config ENABLE_ZIGBEE_FOTA_MAX_HW_VERSION
+	bool "Enable Zigbee OTA maximum hw version"
+
+config ZIGBEE_FOTA_MAX_HW_VERSION
+	hex "Zigbee OTA maximum hw version"
+	default 0
+	range 0x00 0xFF
+	depends on ENABLE_ZIGBEE_FOTA_MAX_HW_VERSION
+
+module=ZIGBEE_FOTA
+module-dep=LOG
+module-str=Zigbee FOTA
+source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
```

```suggestion
source "$(ZEPHYR_BASE)/subsys/logging/Kconfig.template.log_config"
```

https://docs.zephyrproject.org/latest/guides/kconfig/extensions.html#kconfig-extensions

> The recommended syntax for referencing environment variables is $(FOO) rather than $FOO.

---

fixed!